backend/Api/Consultants/ConsultantController.cs

```diff
@@ -34,7 +34,9 @@ public ConsultantController(ApplicationContext context, IMemoryCache cache)
         var selectedYear = selectedYearParam ?? DateTime.Now.Year;
         var selectedWeekNumber = selectedWeekParam ?? DateService.GetWeekNumber(DateTime.Now);
         var selectedWeek = new Week(selectedYear, selectedWeekNumber);
-        var consultants = GetConsultantsWithAvailability(orgUrlKey, selectedWeek, numberOfWeeks)
+        var weekSet = DateService.GetNextWeeks(selectedWeek, numberOfWeeks);
+
+        var consultants = GetConsultantReadModels(orgUrlKey, weekSet)
             .Where(c =>
                 includeOccupied
                 || c.IsOccupied
```

Comment:
Dette har ingen ting med denne PR-en å gjøre egentlig, men skal det ikke være includeOccupied || !c.IsOccupied? For nå får man enten alle, eller kun de som er opptatt 🤔